tests/Rector/Contrib/Symfony/Controller/AddFlashRector/Correct/correct2.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php declare (strict_types=1);
+
+use Symfony\Component\HttpFoundation\Request;
+
+class AnyClass
+{
+    public function someAction(Request $request)
+    {
+        $request->getSession()->getFlashBag()->add('success', 'message');
```

Comment:
Shouldn't it be `$this->addflash('success', 'something');` as is a correct file?

Reply:
No, its not a Controller.